src/cmds/fs/more.c

```diff
@@ -28,29 +28,31 @@ static void screen(FILE *fp) {
 		columns = MAX_SCREEN_WIDTH - 1;
 	}
 	lines = std->endy - std->begy;
-	endwin();
+	//endwin();
 
 	while (1) {
 		for (x = 0; x < lines - 1; x++) {
 
 			for (y = 0; y < columns; y++) {
-				buff[y] = getc(fp);
-				switch ((int) buff[y]) {
-				case EOF:
+				if(fread(&buff[y],1,1,fp)==1){
```

Comment:
Please follow our [code style](https://github.com/embox/embox/wiki/Code-Style), you're missing some spaces here.